GenderPayGap.WebUI/Views/Organisation/ConfirmRemove.cshtml

```diff
@@ -1,21 +1,51 @@
-﻿@model GenderPayGap.WebUI.Models.Organisation.RemoveOrganisationModel
+﻿@using GovUkDesignSystem.GovUkDesignSystemComponents
+@using GovUkDesignSystem
+@model GenderPayGap.WebUI.Models.Organisation.RemoveOrganisationModel
 @{
     Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
-    ViewBag.Title = "Confirm removal of organisation - Gender pay gap service";
+    ViewBag.Title = "Confirm removal of employer - Gender pay gap service";
     var controller = ViewData["Controller"] as BaseController;
 
     long userId = Encryption.DecryptQuerystring(Model.EncUserId).ToInt64();
     bool userRemovingSelf = controller.CurrentUser.UserId == userId && controller.IsImpersonatingUser == false;
     string userNamePostFix = userRemovingSelf ? "You" : "";
 }
+
+@section BeforeMain {
+    @{
+        var crumbs = new List<CrumbViewModel>
+        {
+            new CrumbViewModel
+            {
+                Text = "Manage Employers",
+                Href = Url.Action("ManageOrganisationsGet", "ManageOrganisations")
+            },
+            new CrumbViewModel
+            {
+                Text = Model.OrganisationName,
+                Href = Url.Action("ManageOrganisationGet", "ManageOrganisations", new {encryptedOrganisationId = Model.EncOrganisationId})
+            },
+            new CrumbViewModel
+            {
+                Text = "Remove user"
+            }
+        };
+    }
+
+    @(Html.GovUkBreadcrumbs(new BreadcrumbsViewModel
+    {
+        Crumbs = crumbs
+    }))
+}

```

Comment:
The crumbs seem to be the same for all the pages - the only thing that differs is the text for the last one. I'd add a separate component and use it everywhere

Reply:
Can't fully pull all the repeated content into the same partial, but have pulled as much as possible, I guess it's slightly better?